tests/scenario_quality_test_stock_out.rst

```diff
@@ -0,0 +1,179 @@
+========================
```

Comment:
I think you can merge this test with scenario_quality_test_stock_lot.rst

Seems a lot of scenarios/tests for such a few lines of code.

Reply:
We need to ensure that extras depend works properly

Reply:
finally i merge stock_shipment_in.rst and stock_shipment_out.rst